test.sh

```diff
@@ -1,20 +1,33 @@
 #!/bin/bash
 
-# ./test.sh                bring up, make database, test, bring down
+# UNIT TESTS
+# ./test.sh                build unit test containers, bring up, make database, test, bring down
 # for development:
-# ./test.sh -u             bring up background and load database if needed
-# ./test.sh [params]       run tests, passing optional params to inner test
-# ./test.sh -s             stop test containers without removing
-# ./test.sh -d             clean test containers
+# ./test.sh -u             build unit test containers, bring up background and load database if needed
+# ./test.sh [params]       run unit tests, passing optional params to inner test
+# ./test.sh -s             stop unit test containers without removing
+# ./test.sh -d             clean unit test containers
+
+# FRONTEND TESTS
+# ./test.sh frontend       run frontend tests
```

Comment:
Can we call this `./test.sh -fe`, I don't wanna type `frontend` everytime

Reply:
Sure. Thats too long :p